graphql2/schema.graphql

```diff
@@ -845,6 +845,8 @@ type EscalationPolicy {
 
   assignedTo: [Target!]!
   steps: [EscalationPolicyStep!]!
+
+  notices: [Notice!]
```

Comment:
Same as elsewhere, since it's an output, we can make it "required".

It's only included if asked for, but if asked for you should get an empty array if there are none.

```suggestion
  notices: [Notice!]!
```